inference-engine/src/transformations/src/transformations/low_precision/disable_convert_constant_folding_on_const_path.cpp

```diff
@@ -26,7 +26,7 @@ ngraph::pass::DisableConvertConstantFoldingOnConstPath::DisableConvertConstantFo
 
     ngraph::matcher_pass_callback callback = [=](ngraph::pattern::Matcher & m) -> bool {
         const auto& opsMap = m.get_pattern_value_map();
-        const auto convert = opsMap.find(matcherConvert)->second.get_node()->shared_from_this();
+        const auto convert = opsMap.at(matcherConvert).get_node_shared_ptr();
```

Comment:
Is it expected that at can throw an exception?
```suggestion
        if (opsMap.find(matcherConvert) == opsMap.end())
            return false;
        const auto convert = opsMap[matcherConvert].get_node_shared_ptr();
```

Reply:
Yes, otherwise it will fail during `convert` usage which will be null_ptr.

Reply:
Moreover you shouldn't change `get_pattern_value_map`.